### ISSUE TITLE
conflict with CommandBook commands  ; conflict with MobHealth command

### ISSUE THREAD

**Issue by meoso** [Rocologo/MobHunting#73]
I had to replace config.yml `cmd` statments that contained `give` with `minecraft:give` because [CommandBook](http://dev.bukkit.org/bukkit-plugins/commandbook/) also has a give with a different syntax

I believe due to this same issue, the command for `mobhunt head give does` not work correctly, but i cannot fix with `minecraft:give`:
```
>mobhunt head give daddy husk
[10:15:53 INFO]: Too many arguments.
[10:15:53 INFO]: /give [-do] [-d] <target> <item[:data]> [amount]
>mobhunt head minecraft:give daddy husk
[10:16:13 INFO]: Usage: 
    /mobhunt head give [toPlayername] [playername|mobname] [displayname] [amount] - to give a head
    /mobhunt head rename [new displayname] - to rename the head in players name
```

[MobHealth](http://dev.bukkit.org/bukkit-plugins/mobhealth/) also uses `/mh`, I was able to replace config.yml cmd's `mh` with `mobhunt`.

**Comment by Rocologo** [comment]
I will install mobhealth on my test server ans see if i can solve the conflict.

**Comment by meoso** [comment]
I'm less concerned with MobHealth as it's an easy fix by replacing the command.  I'm more concerned with CommandBook + `mobhunt head`, as i cannot fix it.

**Comment by Rocologo** [comment]
I will look at it i just dont have access to my development laptop these days.

**Comment by Rocologo** [comment]
I finally got some time to look at this. I have tried to force MobHunting to use minecraft:give and I have changed mh to mobhunt in default config.yml file. 

Please help me at test newest build 77 (or newer) to see if it solves your problem.

I have NOT had time to install CommandBook yet, but I hope that me forcing the plugin to use minecraft:give fixes this problem too.

please enable /mh debug before running the command, this way you will be able to see the console command i send to the server.

**Comment by Rocologo** [comment]
I think it is solved in V3.3.6 - I forces my plugin to use minecraft:give which seems to remove the conflict with CommanBook.

**Comment by meoso** [comment]
Thanks for continued support!  First-Rate; Outstanding.
